recipes/catch2/2.x.x/conandata.yml

```diff
@@ -5,3 +5,6 @@ sources:
   2.11.0:
     url: https://github.com/catchorg/Catch2/archive/v2.11.0.tar.gz
     sha256: b9957af46a04327d80833960ae51cf5e67765fd264389bd1e275294907f1a3e0
+  2.11.1:
```

Comment:
You need to add this versio to the config.yml file as well, otherwise it will not be built